CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ set(USE_RCCL_DEFAULT OFF)
 option(USE_REDIS "Support using Redis for rendezvous" ${USE_REDIS_DEFAULT})
 option(USE_IBVERBS "Support ibverbs transport" ${USE_IBVERBS_DEFAULT})
 option(USE_NCCL "Support using NCCL for local collectives" ${USE_NCCL_DEFAULT})
+
 option(USE_RCCL "Support using RCCL for local collectives" ${USE_RCCL_DEFAULT})
```

Comment:
I meant the newline separating the options from the build type.

Reply:
oh that makes more sense! fixed